understanding/21/character-key-shortcuts.html

```diff
@@ -9,43 +9,39 @@
 		<h1>Understanding Character Key Shortcuts</h1>
 		<section id="intent">
 			<h2>Intent</h2>
-<p>Character key shortcuts work well for many keyboard users, but are inappropriate and frustrating for speech input users &#8212; whose means of input is strings of letters &#8212; and for keyboard users who are prone to accidentally hitting keys.
-As a result, users must be able to turn off or reconfigure shortcuts made up of a single character key, or two or more successive character keys.
+<p>The intent of this Success Crition is to reduce accidental activation of keyboard shortcuts. Character key shortcuts work well for many keyboard users, but are inappropriate and frustrating for speech input users &#8212; whose means of input is strings of letters &#8212; and for keyboard users who are prone to accidentally hit keys.
+To rectify this issue, authors need to allow users to turn off or reconfigure shortcuts that are made up of only one or more character keys.
```

Comment:
"made up of only character keys." ?

Reply:
"made up of only character keys" works for me

Reply:
adopted in PR